src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/FileDialog.cs

```diff
@@ -624,14 +613,14 @@ private bool ProcessFileNames()
                             // of the filename in s.
 
                             string newFilename;
-                            if (((ReadOnlySpan<char>)extensions[j]).IndexOfAny('*', '?') != -1)
+                            if (extension.AsSpan().IndexOfAny('*', '?') != -1)
```

Comment:
Maybe even
```suggestion
                            if (extension.AsSpan().ContainsAny('*', '?'))
```
?

Reply:
Yeah this can totally be `ContainsAny`. Thanks, will address.